src/test/java/teammates/test/cases/action/InstructorFeedbackAddActionTest.java

```diff
@@ -219,11 +223,13 @@ protected InstructorFeedbackAddAction getAction(String... params) {
         return (InstructorFeedbackAddAction) gaeSimulation.getActionObject(getActionUri(), params);
     }
 
-    protected String getPageResultDestination(
-            String parentUri, String courseId, String fsname, String userId, boolean isError) {
+    protected String getPageResultDestination(String parentUri, String courseId, String fsname,
+            boolean isToBeLoadedInEditMode, String userId, boolean isError) {
```

Comment:
@whipermr5 I will need to modify the `getIntructorFeedbackLink` because the test fails unless the URL generated from the test helper method and the action destination are the same. See [this line](https://github.com/TEAMMATES/teammates/pull/7615/files/e28f25633cd3a7663cc6cca9c83d7f5a25192b84#diff-64b656ee077cfb7255c4a643b1d83b5fR57).